specs/WindowControlsOverlayConfiguration.md

```diff
@@ -0,0 +1,156 @@
+interface ICoreWebView2WindowControlsOverlaySettings : IUnknown {
+    /// Gets the `Height` property.
+  [propget] HRESULT Height([out, retval] UINT32* value);
+
+
+  /// The `Height` property in raw screen pixels, allows you to set the height of the overlay and
+  /// title bar area. Defaults to 48px. There is no minimum height restriction for this API,
+  /// so it is up to the developer to make sure that the height of your window controls overlay
+  /// is enough that users can see and interact with it. We recommend using GetSystemMetrics(SM_CYCAPTION)
+  // as you minimum height.
+  /// 
+  [propput] HRESULT Height([in] UINT32 value);
+
+
+  /// Gets the `IsEnabled` property.
+  [propget] HRESULT IsEnabled([out, retval] BOOL* value);
+
+
+  /// The `IsEnabled` property allows you to opt in to using
+  /// the WebView2 window controls overlay. Defaults to `FALSE`.
+  /// 
+  /// When this property is `TRUE`, WebView2 will draw its own minimize, maximize,
+  /// and close buttons on the top right corner of the Webview2. 
+  /// 
+  /// When using this you should configure your app window to not display its default
+  /// window control buttons. You are responsible for creating a title bar for your app
+  /// by using the available space to the left of the controls overlay. In doing so, 
+  /// you can utilize the [IsNonClientRegionSupportEnabled](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2settings9?view=webview2-1.0.2739.15)
+  /// API to enable draggable regions for your custom title bar.
+  /// 
+  /// The Overlay buttons will sit on top of the HTML content, and will prevent mouse interactions
+  /// with any elements directly below it, so you should avoid placing content there. 
+  /// To that end, there are four [CSS environment vairables](https://developer.mozilla.org/en-US/docs/Web/API/Window_Controls_Overlay_API#css_environment_variables) 
+  /// titlebar-area-x, titlebar-area-y, titlebar-area-width defined to help you 
+  /// get the dimensions of the available titlebar area to the left of the overlay.
+  /// Similarly the navigator object wil contain a [WindowControlsOverlay property](https://developer.mozilla.org/en-US/docs/Web/API/WindowControlsOverlay)
+  /// which can be used to get the titlebar area as a rect, and listen for changes
+  /// to the size of that area.
+  ///
+  [propput] HRESULT IsEnabled([in] BOOL value);
+
+  /// Gets the `TitleBarBackgroundColor` property.
+  [propget] HRESULT TitleBarBackgroundColor([out, retval] COREWEBVIEW2_COLOR* value);
+
+  /// The `TitleBarBackgroundColor` property allows you to set a background color
+  /// for the overlay. Based on the background color you choose, Webview2 
+  ///will automatically calculate a foreground and hover color that will
+  /// provide you the best contrast while maintaining accessibility.
+  /// Defaults to #f3f3f3. This API supports transparency.
```

Comment:
Document that it does not support changing the default color or default height for OS theming (dark/light mode and high contrast) and height.